src/content/ja/2019/accessibility.md

```diff
@@ -0,0 +1,418 @@
+これらすべて、その誰かが画面を見ることができない理由の説明になります。
+誰もが状況により障害を持ちうるため、Webページのアクセシビリティを改善することは、あらゆる状況ですべてのユーザーの体験を向上させることに繋がります。
+
+[Web Content Accessibility Guidelines](https://www.w3.org/WAI/WCAG21/quickref/) (WCAG)はWebサイトの利便性を向上する方法についてのアドバイスが纏められています。
+このガイドラインを分析の基礎に使いました。
+しかし、ほとんどの場合においてWebサイトのアクセシビリティをプログラムによって分析するのは非常に困難です。
+たとえば、Webプラットフォームは機能的には同じ結果となる複数の方法を提供しており、それを実現するための基盤となるコードはまったく別物になる場合があります。
+したがって、私達の分析結果はWebアクセシビリティ全体の単なる概算でしかありません。
+
+私達はもっとも興味深い洞察を4種類のカテゴリに分類しました。
+それは読みやすさ、Web上のメディア、ページナビゲーションのしやすさ、補助技術との互換性です。
+
+テスト中にデスクトップとモバイルの間でアクセシビリティに大きな違いは見つかりませんでした。
+この結果で提示されているメトリックは、とくに明記していない限りはデスクトップの分析結果です。
+
+## 読みやすさ
+
+Webページの主な目的はユーザーの興味を引くコンテンツを配信することです。
+このコンテンツはビデオや画像の組み合わせなどありますが、ほとんどの場合、シンプルなページ上のテキストです。
+テキストコンテンツが読者にとって読みやすいことは、とても重要です。
+訪問者がWebページを読めない場合、訪問者はWebページに興味を持つことがなくなり、最終的には離脱してしまうでしょう。
+この節ではサイトが苦労するであろう3つの分野を見ていきます。
+
+### 色のコントラスト
+
+あなたのサイトの訪問者が完璧な内容を見ることができない、さまざまな可能性があります。
+訪問者は色覚多様性を持ち、フォントと背景色を区別できない場合があります（ヨーロッパ系の[男性12人に1人、女性200人に1人](http://www.cvrl.org/people/stockman/pubs/1999%20Genetics%20chapter%20SSJN.pdf)）。
+おそらく、彼らは太陽の下で画面の明るさを最大にして読んでいるため、視力を著しく残っているのでしょう。
+もしくは年をとってしまい、彼らの目が以前と同じように色を区別できなくなったのでしょう。
+
+このような条件下であっても、あなたのWebサイトが確実に読めるようにするため、テキストと背景で十分な色のコントラストがあることを確認することは重要です。
+
+<figure>
+  <a href="/static/images/2019/09_Accessibility/example-of-good-and-bad-color-contrast-lookzook.png">
+    <img alt="図1.色のコントラストが不十分なテキストの表示例 LookZook提供" aria-labelledby="fig1-caption" aria-describedby="fig1-description" src="/static/images/2019/09_Accessibility/example-of-good-and-bad-color-contrast-lookzook.png" width="600">
+  </a>
+  <div id="fig1-description" class="visually-hidden">オレンジとグレー4色のボックスに白いテキストがあり、2列に並んでいます。左の列は、色の薄い、#FCA469と書かれたオレンジ色の背景色があります。右の列は、推奨と表示されており、オレンジ色の背景色に#F56905と書かれています。書く列の上のボックスには白いテキスト#FFFFFFにオレンジ色の背景で、下のボックスは灰色の背景に白いテキスト#FFFFFFとなっています。灰色の背景は、オレンジ色をグレースケールにしたものです。LookZook提供
+</div>
+  <figcaption id="fig1-caption">図1.色のコントラストが不十分なテキストの表示例 LookZook提供</figcaption>
+</figure>
+
+すべてのテキストに十分な色のコントラストが適用されているサイトは22.04％のみでした。
+これは言い換えると、5つのサイトのうち4つは背景に溶け込んで読みにくいテキストを持っていると言うことです。
+
+<aside class="note">注意：画像中のテキストは分析できていないため、ここで報告されているメトリックはカラーコントラストテストに合格したWebサイトの総数の上限でしかありません。</aside>
+
+### ズーミングとページのスケーリング
+
+[読みやすいフォントサイズ](https://accessibleweb.com/question-answer/minimum-font-size/)や[ターゲットサイズ](https://www.w3.org/WAI/WCAG21/quickref/#target-size)を使うことで、ユーザーがWebサイトを読んだり操作するのを手助けできます。
+しかし、このガイドラインに対して完全に準拠しているWebサイトですら、訪問者一人ひとりの特定のニーズを満たすことはできません。
+これがピンチズームやスケーリングなどのデバイスによる機能が非常に重要となる理由です。ユーザーが貴方のページを微調整できるようにして彼らのニーズを満たします。
+また、小さなフォントやボタンが使われて操作が非常に難しいサイトであっても、ユーザーにそのサイトを使う機会を与えることができます。
+
+まれですが、スケーリングの無効化が許容される場合はあります。それは問題となるページがタッチコントロールを使ったWebベースのゲームなどの場合です。
+このような場合、有効にしてしまうとプレイヤーがゲームで2回タップをするたびにプレイヤーのスマホがズームインやズームアウトしてしまい、皮肉なことに操作できなくなってしまいます。
+
+なので、開発者は[メタビューポートタグ](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag)で次の2つのプロパティのどちらかを設定することで、この機能を無効化できます。
+
+1. `user-scalable`を`0`か`no`に設定
+
+2. `maximum-scale`を`1`もしくは`1.0`などに設定
```

Comment:
I agree that these attributes should be left in English (as that is how they appear in the HTML) but should we add a Japanese translation of what they say as well?

Reply:
Indeed, this was wrong because it was code.

Reply:
@bazzadp 
Probably not.

Rather than that, I noticed that I need to return the MDN of the link to Japanese.
users can understand the meaning by looking at it.

Reply:
Currently, the URL to `https://developer.mozilla.org` seems to include all the "en-US" paths.
However, `https://developer.mozilla.org` will display the page in the user's language unless you include a location.

Should all `https://developer.mozilla.org/en-US/` be changed to `https://developer.mozilla.org/` here?

Reply:
I think they should be changed to https://developer.mozilla.org/ja to explicitly go to Japanese version when coming from this Japanese translation (in the same way that we explicitly go to US version when coming from English version).

I see you did that for the first link on line 56 but not the rest.

And BTW it says that at https://github.com/HTTPArchive/almanac.httparchive.org/issues/586#issue-533812318:
> Links to French version of resources if available (I'm thinking about MDN or Wikipedia, for example)
Though will update this to Japanese! They were copied from French as examples. 😀

Reply:
@bazzadp 
I'm sorry, I was overlooked.
I will fix it here soon!